make_image.sh

```diff
@@ -406,6 +410,10 @@ install_file etc/systemd/system/installer_backend.service
 install_file boot/efi/installer.ini
 chroot ${target}/ systemctl enable installer_backend
 
+notify installing go build tools
+apt install -y golang-go
```

Comment:
Can you move these up to line ~20 where other dependencies are being installed, please. After that I think this is ready to be merged.

Reply:
Ah, you mean just around where I added the `npm` tools? I moved it there and amended it to the commit

Reply:
No, I mean adding both npm and golang to the list of packages installed with apt on line 22 of make_image.sh

Reply:
Sorry, I misread. I moved the additions to that line.